src/Tools/dotnet-counters/KnownData.cs

```diff
@@ -28,7 +28,7 @@ private static IEnumerable<CounterProvider> CreateKnownProviders(string runtimeV
                 "0xffffffff", // Keywords
                 "5", // Level 
                 new[] { // Counters
-                    new CounterProfile{ Name="cpu-usage", Description="Percentage of time the process has utilized the CPU (%)", SupportedVersions=new[] { net30, net31, net50 } },
+                    new CounterProfile{ Name="cpu-usage", Description="The percent of process' CPU usage relative to all of the system CPU resources", SupportedVersions=new[] { net30, net31, net50 } },
```

Comment:
Should we give an example that specifies whether 100% == 1 core or 100% == all cores?

Reply:
I think “all system resources” already pretty clearly indicates it refers to all cores.

Reply:
Could we specify the range of values then, e.g., "[0,100]%"?  That makes it very explicit.  Other tools call it a "percent" as well even though the value goes all the way up to `N * 100%`.